hera_cal/xtalk_filter.py

```diff
@@ -91,7 +91,8 @@ def run_xtalk_filter(self, to_filter=None, weight_dict=None, max_frate_coeffs=[0
 
 
 def load_xtalk_filter_and_write(infilename, calfile=None, Nbls_per_load=None, spw_range=None, cache_dir=None,
```

Comment:
Can `load_xtalk_filter_and_write` and `load_xtalk_filter_and_write_baseline_list` be combined? They have such similar arguments...

Reply:
My guess is that you could make `xtalk_filter_and_write` a wrapper of `load_xtalk_filter_and_write_baseline_list`.